xgapp-access-points/src/main/xgapp/application-resources/queries/geonames-meta.rq

```diff
@@ -26,7 +24,9 @@ SELECT ?concept ?type ?latitude ?longitude ?ancestors ?population ?wikipediaLink
         ?concept geo:countryCode ?country.
         FILTER(?type = geo:A.ADM1 || ?type = geo:A.ADM1H
```

Comment:
`FILTER (?type in (geo:A.ADM1, geo:A.ADM1H, ...) || ?country in ("AL", "AT" ...)) .`
This variant seems faster for other queries.